fe/fe-core/src/main/java/com/starrocks/scheduler/TaskManager.java

```diff
@@ -607,4 +607,7 @@ private static class SerializeData {
         public List<TaskRunStatus> runStatus;
     }
 
+    public Map<String, Task> getNameToTaskMap() {
```

Comment:
if expose nameToTaskMap , There may be many unexpected problems, such as getting this and deleting the task, etc.